src/drawscene.cpp

```diff
@@ -377,10 +377,10 @@ void draw_top_bottom_3d_mode(Camera& camera, bool show_hud,
 	camera.getCameraNode()->setTarget(oldTarget);
 }
 
-void draw_pageflip_3d_mode(Camera& camera, bool show_hud,
-		Hud& hud, video::IVideoDriver* driver,
-		scene::ISceneManager* smgr, const v2u32& screensize,
-		bool draw_wield_tool, Client& client, gui::IGUIEnvironment* guienv,
+void draw_pageflip_3d_mode(Camera &camera, bool show_hud,
+		Hud &hud, video::IVideoDriver *driver,
+		scene::ISceneManager *smgr, const v2u32 &screensize,
+		bool draw_wield_tool, Client &client, gui::IGUIEnvironment *guienv,
```

Comment:
whats with all these pointless code style changes? it doesn't make sense to fix code style in code you didn't touch